### DISCUSSION TITLE
[FEATURE] Save download state when putting app into the background

### DISCUSSION THREAD

**Discussion by klejejs** [immich-app/immich#3325] [Feature Request]
### The bug

When quickly switching apps in iOS or even opening recents view, mobile app loses active state and backups are restarted or stopped if foreground backup is turned off. In noticed this because GPhotos saves state and backups continue after the app is reopened. It is especially annoying if a large video is being uploaded as it restarts the whole progress.
I am not sure, if it is an implementation issue or limitation of Flutter, it would be nice to understand that and fix the problem.

### The OS that Immich Server is running on

N/A

### Version of Immich Server

v1.67.2

### Version of Immich Mobile App

v1.67.2

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Open app
2. Start backup
3. Open recents and click on Immich again
4. Backup is restarted
```

**Comment by alextran1502** [comment]
I think we just didn't write code to specifically handle this scenario

**Comment by klejejs** [comment]
The same happens when notifications are pulled down as well, which actually quite annoying :)
